Backend/dev/test-tool/dashboard/src/components/LoadTestModal.css

```
@@ -0,0 +1,776 @@
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-top: 1px;
+}
+
+.lt-step-error {
+  color: #ef4444;
+  font-size: 9px;
+  margin-top: 2px;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)